core/src/main/java/com/datastax/oss/driver/api/core/connection/ReconnectionPolicy.java

```diff
@@ -49,9 +51,21 @@
   @NonNull
   ReconnectionSchedule newNodeSchedule(@NonNull Node node);
 
-  /** Creates a new schedule for the control connection. */
+  /**
+   * Creates a new schedule for the control connection.
+   *
+   * @param isInitialConnection whether this schedule is generated for the driver's initial attempt
+   *     to connect to the cluster.
+   *     <ul>
```

Comment:
I think we should avoid long texts like this one inside javadoc tags. With the default CSS style, tag texts are rendered with a fixed-size font and all the formatting is lost:

![image](https://user-images.githubusercontent.com/463876/50488421-9dbdc380-09e9-11e9-963b-b712ba2c7480.png)

Reply:
I hadn't noticed that. I'll move those explanations to the main description then.